core/src/main/java/org/mapfish/print/parser/MapfishParser.java

```diff
@@ -134,10 +135,10 @@ public void parse(final boolean errorOnExtraProperties, final PObject requestDat
         }
     }
 
-    private void checkForExtraProperties(final boolean errorOnExtraProperties, final Class<?> paramClass,
+    private static void checkForExtraProperties(final boolean errorOnExtraProperties, final Class<?> paramClass,
                                          final PObject layer, final String[] extraPropertyToIgnore) {
         final Collection<String> acceptableKeyValues = Sets.newHashSet();
-        for (String name : getAttributeNames(paramClass, FILTER_NON_FINAL_FIELDS)) {
+        for (String name : getAttributeNames(paramClass, FILTER_NON_FINAL_FIELDS::test)) {
```

Comment:
What's this ::test?

Reply:
Was a method reference, but not needed. So I've removed it.